internal/ir/xds.go

```diff
@@ -473,6 +477,9 @@ type StringMatch struct {
 	Suffix *string
 	// SafeRegex match condition.
 	SafeRegex *string
+	// Distinct match condition.
+	// Used to match any and all possible unique values encountered within the Name field.
+	Distinct bool
```

Comment:
The godoc for `StringMatch` should be updated accordingly.

Reply:
thanks for catching this, done